packages/angular_devkit/core/src/virtual-fs/host/memory.ts

```diff
@@ -36,15 +36,16 @@ import {
 
 export interface SimpleMemoryHostStats {
   readonly content: FileBuffer | null;
+  inspect(): string;
 }
 
 export class SimpleMemoryHost implements Host<{}> {
-  protected _cache = new Map<Path, Stats<SimpleMemoryHostStats>>();
+  protected _cache: Map<Path, Stats<SimpleMemoryHostStats>> = new Map();
   private _watchers = new Map<Path, [HostWatchOptions, Subject<HostWatchEvent>][]>();
```

Comment:
I assume this wasn't updated because it's private and won't show up in typings? Should we favor style consistency over the strict requirements (no strong preference either way, just wondering)?

Reply:
Yes, since that is private it's not an issue from an isolated declarations perspective.
Good question, though. For this PR, I went through each error and fixed individually. Since that was not an error, it was not modified.  In general, I do favor consistency. However, we would probably need some form of lint check to both catch all these cases and prevent backsliding.  This may be a good followup task.